bcgov/HOUS-permit-portal
# Feature/bphh 871/invite submitter

## Description

Provides a more descript error message when a user whose email is alreday taken is invited.

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-871

## Steps to QA

log in as review manager
try to invite any email that already exists (submitter@example.com)
error message should appear along with error tag
